### PR TITLE
Replaced hard coded ports with dynamically assigned values from set_env file

### PR DESCRIPTION
**Pull request by okhleif-10**
## Description

Removed all instances of hard coded ports in MMQnA and replaced them with values set from the set_env file in GenAIExamples.
Linked with [This GenAIExamples PR](https://github.com/mhbuehler/GenAIExamples/pull/34)

## Issues

N/A

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [x] Others (enhancement, documentation, validation, etc.)

## Dependencies

N/A

## Tests

Tested by setting env vars, building, composing, and verifying logs.

### CODE REVIEW

File: comps/lvms/llava/dependency/llava_server.py [hunk 1]
```diff
@@ -201,7 +202,7 @@ async def generate(request: Request) -> Response:  # FIXME batch_size=1 for now
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument("--host", type=str, default="0.0.0.0")
-    parser.add_argument("--port", type=int, default=8399)
+    parser.add_argument("--port", type=int, default=os.getenv("LLAVA_SERVER_PORT", 8399))
```
Comment (dmsuehir) [inline comps/lvms/llava/dependency/llava_server.py:205]:
```suggestion
    parser.add_argument("--port", type=int, default=int(os.getenv("LLAVA_SERVER_PORT", 8399)))
```

File: comps/asr/whisper/asr.py [hunk 2]
```diff
@@ -48,6 +48,7 @@ async def audio_to_text(audio: Base64ByteStrDoc):
 
 
 if __name__ == "__main__":
+    whisper_port = int(os.getenv("WHISPER_PORT", 7066))
     asr_endpoint = os.getenv("ASR_ENDPOINT", "http://localhost:7066")
```
Comment (dmsuehir) [inline comps/asr/whisper/asr.py]: Shouldn't this now use `whisper_port` in the default value for `ASR_ENDPOINT`?

File: comps/asr/whisper/dependency/whisper_server.py [hunk 3]
```diff
@@ -106,7 +106,7 @@ async def audio_transcriptions(
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument("--host", type=str, default="0.0.0.0")
-    parser.add_argument("--port", type=int, default=7066)
+    parser.add_argument("--port", type=int, default=os.getenv("WHISPER_PORT", 7066))
```
Comment (dmsuehir) [inline comps/asr/whisper/dependency/whisper_server.py:109]:
```suggestion
    parser.add_argument("--port", type=int, default=int(os.getenv("WHISPER_PORT", 7066)))
```

**Review by dmsuehir — CHANGES_REQUESTED** [review]

**Comment by dmsuehir** [comment]
The Gaudi LVM uses tgi-llava (https://github.com/mhbuehler/GenAIComps/blob/main/comps/lvms/tgi-llava/lvm_tgi.py)

File: comps/asr/whisper/dependency/whisper_server.py (hunk 3 above)
Comment (okhleif-10) [reply comps/asr/whisper/dependency/whisper_server.py:109]: Thought about that but `type=int` takes care of that

File: comps/lvms/llava/dependency/llava_server.py (hunk 1 above)
Comment (okhleif-10) [reply comps/lvms/llava/dependency/llava_server.py:205]: Same as above

**Review by mhbuehler — APPROVED** [review]
LGTM

**Review by dmsuehir — APPROVED** [review]
LGTM, thanks

**Pull request merged**